bridgebot/bidding.py

```diff
@@ -66,21 +66,23 @@ def reset(self):
         self.redoubled = False
         self.consecutive_passes = 0
 
+        self.contract = Contract()
+
     def __increment_player(self):
         self.player = self.player.next_player()
 
     def pass_bid(self):
         if self.bid_index == -1:
             if self.consecutive_passes == 3:  # Passing out
-                self.record[self.player].append(Strains.PASSOUT)
-                self.contract.strain = Strains.NT
+                self.record[self.player].append("PASS")
```

Comment:
Not sure the best place for "PASS" to live. Maybe a new `enum`, but that would create some redundancy